### PR TITLE
Refactor into multiple navbars

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: docs/run/agent/installation.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
-# Fleet Installation
+# Installation
+To get started with the agent, create a new cluster in the Insights UI, e.g. `staging`
```
Comment (makoscafee) [inline docs/run/agent/installation.md:2]: probably link to create cluster?

File: docs/run/agent/installation.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+<div class="mini-img">
+  <img :src="$withBase('/img/new-cluster.png')" alt="new cluster">
+</div>
+
+## Install the Agent
+
+When you create the cluster, you'll be prompted to install the [Insights Agent](https://insights.docs.fairwinds.com/installation/insights-agent/)
```
Comment (makoscafee) [inline docs/run/agent/installation.md:10]: is this link not going to change? `run/agent/installation`?

File: docs/run/agent/about.md [hunk 3]
```diff
@@ -7,27 +7,16 @@ meta:
...
-The Insights Agent is installed and configured using Helm. To see the full list of options,
-you can check out the
-[Insights Agent Helm chart](https://github.com/FairwindsOps/charts/tree/master/stable/insights-agent)
-
-In particular, for any given report type, you can set the following options:
-* `enabled` - this is typically set to `true` by default
-* `schedule` - a cron expression dictating when this report should run. This is typically set to a random minute every hour
-* `timeout` - how long to wait for output, in seconds
-* `resources` - requests and limits for CPU and memory for this report
+To read more about the different report types, see the [Reports Section](/configure/reports/polaris).
```
Comment (makoscafee) [inline docs/run/agent/about.md:19]: mmh should this be linked to `run/agent/configuration` rather than polaris report configuration?

**Review by makoscafee — COMMENTED** [review]
LGTM. just had few questions.

File: docs/run/agent/installation.md (hunk 1 above)
Comment (rbren) [reply docs/run/agent/installation.md:2]: I don't think we can, since the URL includes their org name

File: docs/run/agent/installation.md (hunk 2 above)
Comment (rbren) [reply docs/run/agent/installation.md:10]: Good catch! not sure why it was a full URL

File: docs/run/agent/about.md (hunk 3 above)
Comment (rbren) [reply docs/run/agent/about.md]: good call!

**Review by makoscafee — APPROVED** [review]

**Pull request merged**